common/source/docs/common-fettec-onewire.rst

```diff
@@ -12,10 +17,10 @@ This is a `half duplex serial protocol with a 2Mbit/s Baudrate<https://github.co
 
 This information allows the autopilot to:
 
-- dynamically change the center frequency of the `notch filters used to reduce frame vibration noise in the gyros<https://ardupilot.org/copter/docs/common-imu-notch-filtering.html#dynamic-harmonic-notch>`_
 - log the status of each ESC to the SDCard or internal Flash, for `post flight analysis<https://ardupilot.org/copter/docs/common-logs.html>`_
 - send the status of each ESC to the Ground Station or companion computer for `real-time monitoring<https://ardupilot.org/copter/docs/common-logs.html>`_
-- measure `battery voltage and power consumption<https://ardupilot.org/copter/docs/common-dshot-blheli32-telemetry.html#use-as-battery-monitor>`_
+- optionaly dynamically change the center frequency of the `notch filters used to reduce frame vibration noise in the gyros<https://ardupilot.org/copter/docs/common-imu-notch-filtering.html#dynamic-harmonic-notch>`_
```

Comment:
avoid https links to wiki pages...use the page tag

Reply:
I think I now have this, but how do I link to an html anchor?